idunn/directions/client.py

```diff
@@ -9,6 +9,7 @@
 
 logger = logging.getLogger(__name__)
 
+COMBIGO_SUPPORTED_LANGUAGES = {'en','es','de','fr','it'}
```

Comment:
You have the **_right_** to put whitespace between items you know. :)

Reply:
done :)